site/pages/create-consequence-stops.page.tsx

```diff
@@ -135,6 +148,40 @@ const CreateConsequenceStops = ({
         }
     };
 
+    const getMarkers = () => {
+        const stopsInTable =
+            pageState.inputs.stops && pageState.inputs.stops.length > 0
+                ? pageState.inputs.stops.map((stop) => (
+                      <Marker
+                          key={uniqueId(stop.atcoCode)}
+                          longitude={Number(stop.longitude)}
+                          latitude={Number(stop.latitude)}
+                          anchor="bottom"
+                          color="blue"
+                      />
+                  ))
+                : [];
+        const stopsWithoutStopsInTable = stopOptions
+            .filter((stopToFilter: Stop) =>
+                pageState.inputs.stops && pageState.inputs.stops.length > 0
+                    ? !pageState.inputs.stops.map((s) => s.atcoCode).includes(stopToFilter.atcoCode)
+                    : stopToFilter,
+            )
+            .map((stop) => (
+                <Marker
+                    key={uniqueId(stop.atcoCode)}
+                    longitude={Number(stop.longitude)}
+                    latitude={Number(stop.latitude)}
+                    anchor="bottom"
+                    color="grey"
+                />
+            ));
+
+        const markers = [...stopsInTable, ...stopsWithoutStopsInTable];
+
+        return markers.length > 0 ? markers : null;
```

Comment:
We might need to limit the number of markers we display to 100 or something, I just searched for "street" and the map got pretty slow

Reply:
done